packages/daemon/src/graphql/schema.js

```diff
@@ -40,6 +46,10 @@ const { nodeInterface, nodeField } = nodeDefinitions(
     switch (type) {
       case 'App':
         return ctx.openVault.apps.getByID(typedID)
+      case 'EthHDWalletType':
```

Comment:
Any reason there is a `Type` suffix for this entity and not the others please?
Seems this doesn't match https://github.com/MainframeHQ/js-mainframe/pull/195/files#diff-997d2382d3948ad7f1a5e713e425f2f8R547